packages/strapi-plugin-content-type-builder/services/schema-builder/content-type-builder.js

```diff
@@ -172,6 +172,10 @@ module.exports = function createComponentBuilder() {
             this.unsetRelation(oldAttribute);
           }
 
+          if (Object.keys(oldAttribute).includes('autoPopulate')) {
+            newAttribute.autoPopulate = oldAttribute.autoPopulate;
+          }
```

Comment:
Or here, we can just write:

`newAttribute.autoPopulate = oldAttribute.autoPopulate === false ? false : undefined`